discojs/src/processing/decode.ts

```diff
@@ -0,0 +1,21 @@
+import { PreTrainedTokenizer } from "@xenova/transformers";
+
+
+interface TokenizerOutput {
+  input_ids: number[];
+}

```

Comment:
If you need to define the same interface twice, then you should define it once and import it in the other file!